### DISCUSSION TITLE
Introducing myself, hello there 👋

### DISCUSSION THREAD

**Discussion by shriy-singh** [OpenMined/design#9] [Introductions]
I am Shriya Singh, I am a college undergrad with a background in Computer Science and Engineering. I have been working as a Product Designer and User Researcher (I had started out as a Frontend Developer) for several companies now, and I am keen to become an active open source contributor. But I have always been a little intimidated of larger repositories which I want to change by challenging myself. 
I am hoping to pick up some design tasks and learn new technologies here at OpenMined! 

Have a brilliant day. 
:)

**Comment by Kiaka007** [comment]
Welcome Shriya 👋 👋 
Lovely to have you here ^_^
